templates/appveyor.yml.ejs

```diff
@@ -1,34 +1,18 @@
 environment:
   nodejs_version: "10"
 cache:
-<%_ if (yarn) { _%>
   - '%LOCALAPPDATA%\Yarn -> appveyor.yml'
   - node_modules -> yarn.lock
-<%_ } else { _%>
-  - '%AppData%\npm-cache -> appveyor.yml'
-  - node_modules -> package-lock.json
-<%_ } _%>
 
```

Comment:
This was part of the old oclif codebase that would ask the user to chose the package manager, `plugins:generate` never exposed that to users (default package manager: `yarn`).